src/com/facebook/buck/cli/PublishCommand.java

```diff
@@ -75,6 +75,19 @@
       usage = "Just print the artifacts to be published")
   private boolean dryRun = false;
 
+  @Option(
+      name = "--username",
+      aliases = "-u",
+      usage = "User name to use to authenticate with the server")
+  private String username;
+
+  @Option(
+      name = "--password",
+      aliases = "-p",
+      usage = "Password to use to authenticate with the server")
```

Comment:
Some people don't like to pass in passwords on the command line (it shows up in history).  Is there a way to provide an option to prompt for it too (I'm not sure if buckd is going to rain heavily on this idea).